redhat-director-scripts/rhosp13/puppet/trilio/manifests/horizon.pp

```diff
@@ -1,15 +1,33 @@
 class trilio::horizon (
     $horizon_dir                        = '/etc/openstack-dashboard',
+    $barbican_api_enabled               = hiera('barbican_api_enabled'),
 ){
 
   tag 'triliohorizonconfig'
 
-    file { "${horizon_dir}/local_settings.d":
-        ensure => 'directory',
-    }->
-    file { "${horizon_dir}/local_settings.d/_002_trilio_dashboard.py":
-        ensure  => present,
-        content => template('trilio/_002_trilio_dashboard.py.erb'),
-    }   
+   if $barbican_api_enabled {
+       file_line { "ENABLE OPENSTACK_ENCRYPTION_SUPPORT":  
+           ensure  => present,
+           path   => "${horizon_dir}/local_settings",
+	   line   => 'OPENSTACK_ENCRYPTION_SUPPORT = True',
+	   match  => '^OPENSTACK_ENCRYPTION_SUPPORT',
+       }
+   }    
+   else {
+       file_line { "DISABLE OPENSTACK_ENCRYPTION_SUPPORT":  
+           ensure  => present,
+           path   => "${horizon_dir}/local_settings",
+	   line   => 'OPENSTACK_ENCRYPTION_SUPPORT = False',
+	   match  => '^OPENSTACK_ENCRYPTION_SUPPORT',
+       }
+   }
+ 
+
+    file_line { "DISABLE TRILIO_ENCRYPTION_SUPPORT":
+        ensure => present,
+        path   => "${horizon_dir}/local_settings",
+        line   => 'TRILIO_ENCRYPTION_SUPPORT = False',
```

Comment:
when the variable set to True?

Reply:
For RHOSP13 TRILIO_ENCRYPTION_SUPPORT we are not supporting.
Hence in the RHOSP13 TRILIO_ENCRYPTION_SUPPORT Default value is set to the False.

Reply:
please add this in the code documentation.